Kernel/VM/MemoryManager.cpp

```diff
@@ -20,13 +20,11 @@ MemoryManager& MM
     return *s_the;
 }
 
-MemoryManager::MemoryManager()
+MemoryManager::MemoryManager(u32 pagingoffset)
```

Comment:
"paging" and "offset" are two words, so they should be separated by an underscore.
But it's not very obvious what "paging offset" means.
Let's call it something really obvious, like `physical_address_for_kernel_page_tables`